core/src/main/scala/com/databricks/labs/remorph/discovery/queries.scala

```diff
@@ -19,7 +19,8 @@ case class TableDefinition(
     tableFormat: Option[String] = None,
     viewText: Option[String] = None,
     columns: Seq[StructField] = Seq.empty,
-    sizeGb: Int = 0)
+    sizeGb: Int = 0,
+    comments: Option[String] = None)
```

Comment:
Just checking is the ask is to capture only Table Comments, how about Column Comments?

Reply:
@nfx @gueniai let us know if we need to capture the column comments? Now we are only capturing table comments.

Reply:
We discussed this during standup, but just confirming here that we do need column comments as well

Reply:
done